Mathlib/Order/Monotone/Basic.lean

```diff
@@ -238,42 +238,46 @@ theorem strictAntiOn_toDual_comp_iff : StrictAntiOn (toDual ∘ f : α → βᵒ
   Iff.rfl
 #align strict_anti_on_to_dual_comp_iff strictAntiOn_toDual_comp_iff
 
-protected theorem Monotone.dual (hf : Monotone f) : Monotone (toDual ∘ f ∘ ofDual : αᵒᵈ → βᵒᵈ) :=
-  swap hf
+theorem monotone_dual_iff : Monotone f ↔ Monotone (toDual ∘ f ∘ ofDual : αᵒᵈ → βᵒᵈ) := by
+  rw [monotone_toDual_comp_iff, antitone_comp_ofDual_iff]
+alias monotone_dual_iff ↔ Monotone.dual _
 #align monotone.dual Monotone.dual
```

Comment:
I think it would be cleaner to move that below (with the other aliases), could you do that. Also, I've just realized this makes the names not protected anymore, which is annoying, so I asked on [Zulip](https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/protected.20alias/near/379644420) how can we go around that.

Reply:
@ADedecker  I've moved the aliases - haven't seen a reply on Zulip about the `protected` names yet.

Reply:
This feature indeed doesn't exist yet, but it will soon (leanprover/std4#200). For now this doesn't cause any problems so I'll just merge it, we can add back `protected` later.